examples/docker-swarm-mode/converge-hcl/main.hcl

```diff
@@ -0,0 +1,11 @@
+param "docker-group-user-name" {}
```

Comment:
user. The value of this param is the name of the user that should be added to the docker group. 

As a side note, I originally wanted to be able to name it `docker-group-users` and provide a list of user names but I don't think we can do that (yet), correct?